modis_tools/models.py

```diff
@@ -89,7 +90,7 @@ class CollectionExtended(ApiEntryExtended, Collection):
     has_temporal_subsetting: bool
     has_transforms: bool
     has_variables: bool
-    orbit_parameters: dict
+    orbit_parameters: Dict[Any, Any]
```

Comment:
Just for my information: other than being more explicit, is there any benefit to the new typing syntax?

Reply:
Similar question:
* why explicitly type with a `typing.Dict` instead of just `dict`? Can you point to where it recommends this in the migration docs
* is the purpose of `Any` just a placeholder to show that we haven't explicitly typed it yet? From glancing at the downstream code, we definitely have expectations for what the contents of the lists and dicts are, so I'd recommend we add the explicit typing that either now or in a second step

Reply:
I take back my `Any` comment re: doing it now. Adding in `Any` lets you get pylance off your back while still retaining a signal to explicitly type things later